src/meta/meta_service.cpp

```diff
@@ -1078,6 +1080,23 @@ void meta_service::on_notify_stop_split(notify_stop_split_rpc rpc)
                      server_state::sStateHash);
 }
 
+void meta_service::on_query_child_state(query_child_state_rpc rpc)
+{
+    if (!check_status(rpc)) {
+        return;
+    }
+
+    if (_split_svc == nullptr) {
+        derror_f("meta doesn't support partition split");
+        rpc.response().err = ERR_SERVICE_NOT_ACTIVE;
+        return;
+    }
+    tasking::enqueue(LPC_META_STATE_NORMAL,
+                     tracker(),
+                     [this, rpc]() { _split_svc->query_child_state(std::move(rpc)); },
+                     server_state::sStateHash);
```

Comment:
```suggestion
    _split_svc->query_child_state(std::move(rpc));
```
You don't have to enqueue because this is a read operation.